astropy/utils/masked/tests/test_function_helpers.py

```diff
@@ -897,6 +893,22 @@ def test_average(self):
         assert_array_equal(o.unmasked, expected)
         assert_array_equal(o.mask, expected_mask)
 
+    @pytest.mark.parametrize("kwargs", [{}, {"axis": 0}])
+    def test_ptp(self, kwargs):
+        o = np.ptp(self.ma, **kwargs)
+        expected = self.ma.max(**kwargs) - self.ma.min(**kwargs)
+        assert_array_equal(o.unmasked, expected.unmasked)
+        assert_array_equal(o.mask, expected.mask)
+        out = np.zeros_like(expected)
```

Comment:
Probably worth commenting what each test block is doing

```suggestion
        # testing setting via `out`
        out = np.zeros_like(expected)
```

Reply:
There are lots of tests in the file like this. Still, happy to do it, though perhaps once I rebase #15378? Prefer to avoid running CI again...